lib/actions/up.js

```diff
@@ -34,14 +34,21 @@ module.exports = async (db, client) => {
       throw error;
     }
 
-    const { changelogCollectionName, useFileHash } = await config.read();
+    const { changelogCollectionName, useFileHash, saveFileContents } = await config.read();
     const changelogCollection = db.collection(changelogCollectionName);
 
-    const { fileName, fileHash } = item;
+    const { fileName, fileHash, fileContents } = item;
     const appliedAt = new Date();
 
     try {
-      await changelogCollection.insertOne(useFileHash === true ? { fileName, fileHash, appliedAt } : { fileName, appliedAt });
+      const migration = { fileName, appliedAt };
+      if (useFileHash) {
+        migration.fileHash = fileHash;
+      }
+      if (saveFileContents) {
```

Comment:
If we are saving file contents, we include them in the data that is saved in the database